src/Microsoft.DotNet.Web.ProjectTemplates/RazorClassLibrary-CSharp.csproj.in

```diff
@@ -1,11 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.RazorClassLibrary1</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="${MicrosoftAspNetCoreMvcPackageVersion}" />
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
cc @pranavkm @rynowak @dsplaisted - it wasn't clear to me if Microsoft.NET.Sdk.Razor would also imply this framework reference, so I've left it in for now.

Reply:
Right now, the Razor SDK doesn't implicitly add the framework reference.  We can discuss whether it should, I don't really know right now.